src/RenderWebGL.js

```diff
@@ -621,6 +621,60 @@ RenderWebGL.prototype._drawThese = function(
 };
 
 /**
+ * Draw say bubble next to the Drawable
+ * @param {int} drawableID A Drawable ID for draw bubble to
+ * @param {string} text A text for render
+ */
+
+RenderWebGL.prototype.renderBubble = function (drawableID, text) {
+    var  drawable = Drawable.getDrawableByID(drawableID)
+        ,bounds = drawable.getBounds()
+        ,xy = {x: bounds.right + 5,
+               y: bounds.top}; // A point for bubble render
+
+    var canvas = typeof document !== 'undefined' ?
+                 document.createElement('canvas') : null,
+        ctx;
+    if (!canvas) return; //throw new Error('Cannot create canvas!');
+    ctx = canvas.getContext('2d');
+
+    var w = ctx.measureText(text).width,
```

Comment:
The font should be set before any `ctx.measureText`, otherwise your measurement would likely be for a different font.